.gitignore

```diff
@@ -81,3 +81,5 @@ aws-cpp-sdk-core/include/aws/core/SDKConfig.h
 
 #nuget
 *.nupkg
+
+.deps
```

Comment:
i don't think this is necessary if we are building out of the source tree.

Reply:
aws-adk-cpp are fetching and building their dependencies inside their source tree

Reply:
could you be more specific which dependencies are build in the source tree?

Reply:
see this section in the cmake: https://github.com/ceph/aws-sdk-cpp/blob/master/CMakeLists.txt#L148
actually, there is no reason to build inside their source tree. will try to set their parameter to point into our "build" dir

Reply:
i don't really understand how/why `CMAKE_CURRENT_BINARY_DIR` points to the "their source tree".

Reply:
will look into that

Reply:
i think that's because I set `AWS_DEPS_INSTALL_DIR` here: 
https://github.com/prateek2211/ceph/blob/7afaa86626e196a57e6c002e2f4b9ce707ad88f1/src/CMakeLists.txt#L300

Reply:
i could change that to "${CMAKE_SOURCE_DIR}/build/.deps"

Reply:
i think https://github.com/ceph/ceph/blob/master/.gitignore#L64 is good enough.

Reply:
@prateek2211 can you please make sure it is set to: `${CMAKE_CURRENT_BINARY_DIR}`
this will translate to `build/aws-sdk-cpp/`
if this workls, there is no need to use `.deps`

Reply:
yes, I did. `${CMAKE_CURRENT_BINARY_DIR}` points to `build/src/.deps`

Reply:
So, I have removed `.deps`